app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -121,6 +124,58 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+fun infoButton(modifier: Modifier = Modifier, isEnabled: Boolean,
+               onClick: () -> Unit,shape: DepthSurfaceShape = DepthSurfaceShape.Circle) {
+    DepthButton(
+
+        modifier = modifier.size(70.dp),
+        isEnabled = isEnabled,
+        colors = AppButtonColors.Default,
+        onClick = onClick,
+        shape = DepthSurfaceShape.Circle,
+        depth = 10f
+    ) {
+        infoButtonCircle(
+            modifier = Modifier
+                .size(60.dp),
+            color = AppColors.LightGray,
+            shadowColor = AppColors.GrayShadow,
+        )
```

Comment:
Instead of recreating the icon, we can reuse R.drawable.info_icon instead. We can reuse a lot of what this button used to be, and place is here instead.

Reply:
Thanks! Addressed.